src/App.tsx

```diff
@@ -17,18 +73,43 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                onChangeQuery={setQuery}
+                selectedStatus={selectedStatus}
+                onSelectStatus={setSelectedStatus}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length && (
```

Comment:
`todos.length` returns a number and it can be zero. In this case react will render `0` on the page, but you don't want to render anything. You need to transform a number to boolean, because for false react will not render anything at all. One of the ways to do it is like this.
```suggestion
              {Boolean(todos.length) && (
```